CHANGELOG.md

```diff
@@ -11,7 +11,7 @@ and this project adheres to [Semantic Versioning](http://semver.org/)
 - New Search Bar #526
 
 ### Changed
-
+- Position of the legendPanel #633
```

Comment:
Keep in mind that these are our release notes. Something like "LegendPanel can be found in the bottom right now #633" would be better. Otherwise the user has to check the issue to see where it actually went